api/controller/searchController.js

```diff
@@ -0,0 +1,22 @@
+import { dbController } from './dbController';
+import 'dotenv/config';
+
+export const getSongs = async (req,res) => {
+  try {
+    const songName = req.params.song.replace('%20', ' ');
+    const artistName = req.params.artistName;
```

Comment:
You can use the object separation trick (i forgot the name):
```
  const { artistName } = req.params
```
or something like that...